modules/speedtest.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+function speedtest {
+  /usr/bin/speedtest "$@"
```

Comment:
```
root@pattanawadee88:~/cli# ./cli.sh speedtest
/root/cli/modules/speedtest.sh: line 4: /usr/bin/speedtest: No such file or directory
```
we should check if the speedtest is installed

Reply:
you need to install speedtest-cli first, then try again
```
apt install speedtest-cli
```

Reply:
110 image installed speedtest-cli, you can test right now on new image.